utilities/upload_julia.sh

```diff
@@ -17,7 +17,7 @@ UPLOAD_EXTENSIONS=( "tar.gz" )
 
 # Only codesign if we are not on a pull request build.
 # Pull request builds only upload unsigned tarballs.
-if [[ "${BUILDKITE_PULL_REQUEST}" == "false" ]]; then
+#if [[ "${BUILDKITE_PULL_REQUEST}" == "false" ]]; then
```

Comment:
Was this intended? Should it be removed then?

Reply:
That change is in a commit marked "DROP ME", so I think it's just a temp change while debugging this PR.

Reply:
Ah, heh, sorry for the noise.